### ISSUE TITLE
Documentation does not contain chip configuration fields

### ISSUE THREAD

**Issue by Petro31** [piitaya/lovelace-mushroom#238]
**Is your feature request related to a problem? Please describe.**
I just tried to help someone use the template chip and a few other cards in this.  The documentation is lacking and the verbiage for the fields goes against what is used across all of home assistant.

e.g.  Template card uses `primary` and `secondary` as name/label fields.  These are typically referred to as `name` and `secondary`/`label`.

The chips has 0 documentation on how to configure any of them.  If you want to configure them, you need to basically seek help from someone who can read code.

**Describe the solution you'd like**
primary on the template card reads 'primary info...', that should read `primary text...`, same with secondary.  Or possible a longer expanation as to what this is filling in.  You can even change the example image to just show the fields it's templating by using "Primary" and "Secondary" in the chip image at the top of the page.  Or an example that pairs with the image that's currently there.

Each chip on the chips document page needs a table with configuration options like all the cards have.

**Describe alternatives you've considered**
Reading code should never be an option for users, so there is no alternative.

**Additional context**
No

**Comment by piitaya** [comment]
Hi 🙂
The documentation is not complete due to lack of time 😅 
As there is an editor for each option for each card and each chip, we didn't update the documentation because we though it's self explanatory.

But I agree, the documentation must be updated (especially for the template card/chip). We are aware of it and we will do it as soon as we have time.

**Comment by Petro31** [comment]
> 
Yeah, I was just pointing out some things from a help perspective.  I don't have it installed but I was helping a person configure it.  I had to read the code in order to help them because there were no yaml examples or field descriptions.
